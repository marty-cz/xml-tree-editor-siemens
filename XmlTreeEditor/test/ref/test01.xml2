<?xml version="1.0" encoding="ISO-8859-2"?><CATALOG>
  <CD ID="cd01">
    <TITLE>Empire Burlesque</TITLE>
    <ARTIST>Bob Dylan</ARTIST>
    <COUNTRY>USA</COUNTRY>
    <COMPANY>Columbia</COMPANY>
    
    <YEAR>1985</YEAR>
  </CD>
  <CD ID="cd02">
    <TITLE>Hide your heart</TITLE>
    <ARTIST>Bonnie Tylor</ARTIST>
    <COUNTRY>GB</COUNTRY>
    <COMPANY>CBS Records</COMPANY>
    
    <YEAR>1988</YEAR>
  </CD>
  <CD ID="cd03">
    <TITLE>Greatest Hits</TITLE>
    <ARTIST>Dolly Parton</ARTIST>
    <COUNTRY>USA</COUNTRY>
    <COMPANY>RCA</COMPANY>
    
    <YEAR>1982</YEAR>
  </CD>
  <CD ID="cd04">
    <TITLE>Still got the blues</TITLE>
    <ARTIST>Gary More</ARTIST>
    <COUNTRY>GB</COUNTRY>
    <COMPANY>Virgin redords</COMPANY>
    
    <YEAR>1990</YEAR>
  </CD>
</CATALOG>