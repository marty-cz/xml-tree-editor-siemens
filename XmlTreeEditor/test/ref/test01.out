Input xml file "test01.xml" is valid according to schema "schema01.xsd"
